jobserver/templates/org_detail.html

```diff
@@ -34,8 +36,11 @@ <h3>Projects</h3>
     </ul>
     {% else %}
     <p class="mt-5">
-      There are no projects for this Organisation yet, click "Register Project"
-      to create one.
+      {% if is_member %}
+      There are no projects for this Organisation yet, click "Register Project" to create one.
+      {% else %}
+      There are no projects for this Organisation yet.
```

Comment:
s/O/o/!

Reply:
ARG

Genuinely tempted to write a pre-commit hook for this…